src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Alias.kt

```diff
@@ -39,5 +40,14 @@ data class Alias(
         dateOfBirth = alias.dateOfBirth,
         title = alias.title,
       )
+    fun convertEntityToAlias(pseudonymEntity: PseudonymEntity): Alias {
```

Comment:
We have used [this pattern](https://github.com/ministryofjustice/hmpps-person-record/blob/75ef24260f7a521ce958433766b14c6637f941c9/src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Person.kt#L52) elsewhere in the code